catalog/spatial/geocoding/spatial-geocoding-create/src/test/java/org/codice/ddf/spatial/geocoding/create/TestGeoNamesCreator.java

```diff
@@ -33,7 +33,8 @@ public void testNoEmptyFields() {
                 -111.84125,
                 "PPL",
                 236123,
-                "Candler,Candleris,Chandler,Chandlur");
+                "Candler,Candleris,Chandler,Chandlur",
+                "US");
```

Comment:
Yes. The metacard reflects the 3 letter country code using the ISOFormatConverter - but the gazetteer (which this class is testing the results returned from it) stores and returns a 2 letter country code.